src/grt/src/GlobalRouter.tcl

```diff
@@ -404,5 +385,12 @@ proc define_clock_layer_range { layers } {
   }
 }
 
+proc check_pdrev_alpha { alpha } {
```

Comment:
I supposed you could have a separate proc for this but when I did it I just used one if. See the rsz_use_pd2 PR